source/Halibut.TestUtils.CompatBinary.Base/CompatBaseDelegateServiceFactory.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Halibut.ServiceModel;
+
+namespace Halibut.TestUtils.SampleProgram.Base
+{
+    public class CompatBaseDelegateServiceFactory : IServiceFactory
```

Comment:
The interface has changed between v5 and v6 of Halibut so we have to have our own version

Reply:
ugh who did that :p